scripts/install-globals.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-npm install -g gulp jspm typescript tsd
\ No newline at end of file
+npm install -g gulp typescript tsd
```

Comment:
Pytanie, czy to faktycznie potrzebne jest globalnie? Zawsze starałem się korzystać z lokalnych wersji dostępnych w `node_modules/.bin` i było to bardziej przewidywalne w zachowaniu.